components/common/cloud_card.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import './cloud_card.scss';
+
+type CloudCardProps = {
+    children: JSX.Element[] | JSX.Element | string;
+    className: string;
+}
+
+const CloudCard: React.FC<CloudCardProps> = (props: CloudCardProps) => {
```

Comment:
Same comment here about if this there is something cloud specific on this card?

Reply:
there is already a component called Card, I checked it and was too complex for this need, however, since this one was meant only to be a container, maybe call it CardContainer?

Reply:
+1 on `CardContainer`, just want to drop the `cloud` reference here given that this is open to be used by any